lesson2.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Git tutorial</title>
+</head>
+
+<body>
+    Nguyen Van B
```

Comment:
```suggestion
    Nguyen Van C
```
the nay mới đúng ô ơi